src/tests/ftest/util/daos_core_base.py

```diff
@@ -15,6 +15,7 @@
 from general_utils import get_log_file
 from command_utils import CommandFailure
 from agent_utils import include_local_host
+from general_utils import run_pcmd, colate_results
```

Comment:
(pylint-unused-import) Unused run_pcmd imported from general_utils
(pylint-unused-import) Unused colate_results imported from general_utils
(pylint-ungrouped-imports) Imports from package general_utils are not grouped